dts/bindings/sdhc/sdhc.yaml

```diff
@@ -0,0 +1,38 @@
+#
```

Comment:
Is the goal of this file to collect properties which are common to multiple MMC host controllers?
If yes, I believe that it should be moved under `dts/bindings/mmc` and rename to `mmc.yaml`.

Reply:
moved to dts/bindings/mmc/ and rename to mmc-sdhc.yaml.

Reply:
fixed in commit 9dc0d12bfe05f783471e35a3034e4b9276c8e818

Reply:
fixed in commit 6f1e1579db1ecb4ba8cd27c1c0b13b4ae5d5daf0
nxp,imx-usdhc.yaml: specifies nxp usdhc module.
nxp,imx-usdhc-slot.yaml: specifies usdhc slot on board which inherits usdhc.
on board dts:
&usdhc1 {
       status = "ok";
       compatible = "nxp,imx-usdhc-slot";
       pwr-gpios = <&gpio1 5 0>;
       cd-gpios = <&gpio2 28 0>;
};